build-logic/testing/src/main/kotlin/convention.unit-testing.gradle.kts

```diff
@@ -1,3 +1,5 @@
+import gradle.kotlin.dsl.accessors._aefff6f539276b188c30d8843d1e61e9.testImplementation
+import gradle.kotlin.dsl.accessors._aefff6f539276b188c30d8843d1e61e9.testRuntimeOnly
```

Comment:
What is this package? Does it expected?

Reply:
nope, it could lead to accessor problems. let me think if we can defent from it via detekt or something similair

Reply:
https://github.com/avito-tech/avito-android/pull/1131